opendc-model-odc/jpa/src/main/resources/jpa/schema.xml

```diff
@@ -184,13 +184,25 @@
             </many-to-one>
             <many-to-many name="dependencies" target-entity="Task">
                 <join-table name="task_dependencies">
-                    <join-column name="second_task_id"/>
-                    <inverse-join-column name="first_task_id"/>
```

Comment:
These need to be kept `second_task_id` and `first_task_id` to retain compatibility with the database schema

Reply:
Ok I'll change that back.